somafm.el

```diff
@@ -164,6 +176,25 @@ If the favorite file doesn't exist it will be created."
               (delete-char -1)
               (somafm--clear-rest-of-line))))))))
 
+(defun somafm--volume-control (steps direction)
+  "Modify the volume n STEPS in DIRECTION."
+  (let ((steps (if steps steps 1))
+	(direction-key (if (string= "up" direction)
```

Comment:
also believe it would be a bit more idiomatic in this case to make "up" and "down" symbols `'up` `'down` rather than strings